membership-attribute-service/app/services/AttributesFromZuora.scala

```diff
@@ -44,10 +46,8 @@ class AttributesFromZuora(implicit val executionContext: ExecutionContext) exten
     }
 
     val zuoraAttributesDisjunction: DisjunctionT[Future, String, Future[Option[ZuoraAttributes]]] = for {
-      accounts <- EitherT[Future, String, QueryResponse](withTimer(s"ZuoraAccountIdsFromIdentityId", zuoraAccountsQuery(identityId, identityIdToAccountIds), identityId))
-      accountIds = queryToAccountIds(accounts)
-      accountSummaries <- EitherT[Future, String, List[AccountSummary]](getResultIf(accountIds.nonEmpty, "ZuoraGetAccountSummaries", getAccountSummaries(accountIds, identityId, accountSummaryForAccountId), Nil, identityId))
```

Comment:
great, that will be a good improvement in calls made

Reply:
Back to what it was before, where we're pretty content that it is a stable baseline.